lib/tasks/export.rake

```diff
@@ -20,6 +20,7 @@
 
 require 'zip'
 require 'fileutils'
+require 'eml_to_pdf'
```

Comment:
I had to monkey patch to use the correct `wkhtmltopdf` binary:
```ruby
module EmlToPdf
  class Wkhtmltopdf
    class ConversionError < StandardError; end

    def self.convert(input, output_path)
      IO.popen("#{PDFKit.configuration.wkhtmltopdf} --encoding utf-8 --footer-center [page] --footer-spacing 2.5 --quiet - #{output_path} 2>&1", "r+") do |pipe|
        pipe.puts(input)
        pipe.close_write
        output = pipe.readlines.join
        pipe.close
        unless $?.success?
          raise ConversionError, output
        end
      end
    end
  end
```

Reply:
sounds good, I'll add the monkey patch.

Reply:
what if we just `exec("PATH=#{PDFKit.configuration.wkhtmltopdf}:$PATH")` does that work? (im not wise in the path ways)